public/employers/tyson-foods.yml

```diff
@@ -0,0 +1,56 @@
+name: Tyson Foods, Inc.
+aliases:
+  - Aidells
+  - Ball Park
+  - Hillshire Farm
+  - Jimmy Dean
+  - Sara Lee
+  - State Fair
+  - Wright Brand
+shortName: Tyson
+ticker: TSN
+wiki: Tyson Foods
+image: tyson-foods.svg
+officialWebsite: https://www.tysonfoods.com/who-we-are
+industries:
+  - foodProcessing
+location:
+  city: Springdale
+  state: AR
+  country: us
+  international: true
+  wiki: Springdale, Arkansas
+employeesBefore:
+  type: approximately
+  upperBound: 122000
+  year: 2015
+summary: Tyson took proactive and reactive measures to keep employees safe and shut down facilities where there were perceived safety issues. However, it received criticism for its slow response and for keeping non-essential product lines operating.
+citations:
+  - summary: Tyson promised to compensate all employees affected by Waterloo plant closures.
+    positivity: 1
+    type: publication
+    sources:
+      - source: CNN
+        link: https://www.cnn.com/2020/04/22/us/tyson-waterloo-iowa-plant-employees-coronavirus/index.html
+        date: 2020-04-23T01:05:04Z
+  - summary: Tyson closed down a pork plant in Waterloo due to concerns regarding COVID-19; however, officials and staff criticized Tyson for waiting too long to do so.
+    positivity: 0
+    type: publication
+    sources:
+      - source: CNN
+        link: https://www.cnn.com/2020/04/22/us/tyson-waterloo-iowa-plant-employees-coronavirus/index.html
+        date: 2020-04-23T01:05:04Z
+  - summary: Tyson was deemed essential due to being not only involved in food service, but pet food as well, and therefore continued to operate its plants during the pandemic.
```

Comment:
You should check out this article. https://www.bloomberg.com/news/articles/2020-04-29/tyson-foods-helped-create-the-meat-crisis-it-now-warns-against

It explains the meat crisis and how even unions say that it wasn't safe for the workers to work there. You may also mention how the company is having a hard time finding workers.